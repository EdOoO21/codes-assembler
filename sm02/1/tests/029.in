-845 -292 225 730 240 -409 -823 45 899 -296 -196 -150 -962 994 -934 -788 735 -90 335 -540 644 -285 -825 19 -326 878 244 110 -141 712 495 -592 -85 -651 -551 -112 605 -165 380 623 400 152 72 -598 113 380 -135 -389 -431 48 536 -550 676 -159 963 609 -33 56 -706 540 -372 549 -388 202 449 321 -35 -187 681 265
