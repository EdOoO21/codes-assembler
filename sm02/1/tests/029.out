265
681
-187
-35
321
449
202
-388
549
-372
540
-706
56
-33
609
963
-159
676
-550
536
48
-431
-389
-135
380
113
-598
72
152
400
623
380
-165
605
-112
-551
-651
-85
-592
495
712
-141
110
244
878
-326
19
-825
-285
644
-540
335
-90
735
-788
-934
994
-962
-150
-196
-296
899
45
-823
-409
240
730
225
-292
-845
